### ISSUE TITLE
Bosch C Harness

### ISSUE THREAD

**Issue by vanillagorillaa** [commaai/opendbc#1298]
- [x]  which car is this for? 2023 Honda Pilot, 2023 CR-V, 2023 Accord
- [x]  schematic - Push to neo repo once confirmed working
- [x]  beta sent to tester
- [ ]  in shop
- [ ]  Added to openpilot docs

Reported on Discord by user "Kevmaninc". Unknown connector part number. Will have to grab wiring diagram from Honda Techinfo. Unknown camera module. No marking on underside of camera. Will assign proper harness name once more information is obtained.

**Comment by vanillagorillaa** [comment]
https://github.com/commaai/website/commit/01bb7f63ef87290a416773bb1867634f3a8b5d7a

**Comment by shahrk** [comment]
> https://github.com/commaai/website/commit/01bb7f63ef87290a416773bb1867634f3a8b5d7a

Does this mean CRV 2023 should be supported now? The website still doesn't show that it's supported.

**Comment by Kevmaninc** [comment]
That is correct, the CRV 2023 has the same support as Pilot 2023.

**Comment by ba5714** [comment]
What is the latest fork for honda pilot 2023?

**Comment by Kevmaninc** [comment]
> What is the latest fork for honda pilot 2023?

You should just be able to use the main OpenPilot as it should work now. However if you are interested in a SunnyPilot version then you have some options. 

VG Pilot Install:
installer.comma.ai/vanillagorillaa/Pilot_2023

Babyhamsta
installer.comma.ai/Babyhamsta/dev-c3

**Comment by ba5714** [comment]
On the main open pilot, it doesn't work. Maybe a newer version is needed?

**Comment by Kevmaninc** [comment]
Try the links I sent you.

**Comment by ba5714** [comment]
installer.comma.ai/vanillagorillaa/Pilot_2023

This fork stopped working. Why? What is the replacement?

Thanks

**Comment by Mr-Peter-Ng** [comment]
@ba5714 that link is being worked on & updated. It should be back up & running soon.

**Comment by ba5714** [comment]
No, it's still not working.

**Comment by Kevmaninc** [comment]
Are you using the main comma branch? You need to go to discord and check with the community.Sent from my iPhoneOn Apr 17, 2025, at 1:29 PM, ba5714 ***@***.***> wrote:﻿
  No, it's still not working.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>



ba5714 left a comment (commaai/opendbc#1298)
No, it's still not working.

**Comment by ba5714** [comment]
I'm trying to use this one.

installer.comma.ai/vanillagorillaa/Pilot_2023

**Comment by Kevmaninc** [comment]
No, you need to use something else. Try the one below and choose your car
from the list.

installer.comma.ai/kevmaninc/dev-c3

**Comment by smtermini** [comment]
So I just purchased the Bosch C Harness from Comma.Shop it fits perfectly in the 2024 Honda Accord hybrid. Now it seems that the original Comma software doesn’t support the 2024 accord. Anyone know if SunnyPilot, DragonPilot or another fork will work?

**Comment by Kevmaninc** [comment]
Yes, go to Discord and join the community there are lots of forks that work
with that car.
